dev/com.ibm.ws.session.db/resources/com/ibm/ws/session/db/resources/WASSession.nlsprops

```diff
@@ -156,9 +156,9 @@ DatabaseHashMap.selectNoUpdateError.explanation=An exception has been detected w
 DatabaseHashMap.selectNoUpdateError.useraction=If a SQLException has occurred then refer to the appropriate database documentation for your environment. Also, ensure that you have properly configured a data source for the Session Manager.
 
 # -------------------------------------------------------------------------------------------------
-DatabaseHashMapMR.db2LongVarCharErr=SESN8558E: An attempt was made to write more than 2M to the large column.
+DatabaseHashMapMR.db2LongVarCharErr=SESN8558E: An attempt was made to write more than row size limit to the database column.
```

Comment:
suggested rewrite:
SESN8558E: An attempt was made to write more to the database column than is allowed by the value of the rowSizeLimit property.